go/kbtest/kbtest.go

```diff
@@ -234,8 +234,8 @@ func FakeSalt() []byte {
 // the test will not fail until later.
 // `tcX` is a TestContext where device X (the provisioner) is already provisioned and logged in.
 // this function will provision a new device Y inside `tcY`
-// `newDeviceType` is libkb.DeviceTypeMobile or libkb.DeviceTypeDesktop.
-func ProvisionNewDeviceKex(tcX *libkb.TestContext, tcY *libkb.TestContext, userX *FakeUser, newDeviceType string) {
+// `newDeviceType` is keybase1.DeviceTypeV2_MOBILE or keybase1.DeviceTypeV2_DESKTOP.
```

Comment:
same as above, do you think this comment is still useful?

Reply:
as above, not sure, so better err on the side of caution